### PR TITLE
Refine datasource service component error message

### PR DESCRIPTION
**Pull request by dangquangdon**
[DARK-473]

[DARK-473]: https://aiven.atlassian.net/browse/DARK-473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: internal/service/service_component/datasource_service_component.go [hunk 1]
```diff
@@ -135,9 +208,27 @@ func datasourceServiceComponentRead(_ context.Context, d *schema.ResourceData, m
...
 			}
 
-			// check optional kafka_authentication_method search criteria, if not set by a
-			// user match entries without kafka_authentication_method
-			if method, ok := d.GetOk("kafka_authentication_method"); ok {
+			filteredResult = append(filteredResult, c)
+		}
+
+		if len(filteredResult) == 0 {
+			var err_msg string
```
Comment (ivan-savciuc) [inline internal/service/service_component/datasource_service_component.go:215]: Would you mind changing `err_msg` to camel Case for consistency?

Comment (dangquangdon) [reply internal/service/service_component/datasource_service_component.go:215]: ah, yeah of course, it's python habit :sweat:

Comment (ivan-savciuc) [reply internal/service/service_component/datasource_service_component.go]: np :pray:

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**